plugins/kubernetes-backend/src/service/KubernetesProxy.ts

```diff
@@ -104,6 +151,11 @@ export class KubernetesProxy {
 
           res.status(500).json(body);
         },
+        onProxyReq: (proxyReq, req) => {
+          // the kubernetes proxy endpoint expects a header field labeled `Backstage-Kubernetes-Authorization` that will be used to authenticate with the Kubernetes Api. The token provided as a value should be an bearer token for the target cluster.
+          const token = req.header(HEADER_KUBERNETES_AUTH) ?? '';
+          proxyReq.setHeader('Authorization', token);
```

Comment:
Just to double check, this will remove the Backstage token header from the upstream request even if token is `''` right?

Reply:
thats right, if one is provided upstream it will simply be passed along so in the case of a '' it would be passed through, if one isnt provided(in the case of null or undefined) it will default to passing an empty string which the kubeapi will error handle and pass back to the requester.